content/panel_release_1.3.ipynb

```diff
@@ -0,0 +1,1970 @@
+{
```

Comment:
<p>Should <code>PanelCallbackHandler</code> be linked to <a href="https://panel.holoviz.org/reference/chat/PanelCallbackHandler.html" target="_blank">https://panel.holoviz.org/reference/chat/PanelCallbackHandler.html</a></p><p> </p><p>And maybe more specific instructions about <code>call us</code> (inviting them to our Discord / Discourse). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>